dist/index.html

```diff
@@ -7,8 +7,14 @@
 </head>
 
 <body>
+  <div class="button-container" id="main-menu">
+    <button class="menu-btn" id="new-game">New game</button>
+    <button class="menu-btn" id="settings">Settings</button>
+    <button class="menu-btn" id="high-scores">High scores</button>
+    <button class="menu-btn" id="about">About</button>
+  </div>
   <canvas></canvas>
   <script src="main.js"></script>
 </body>
 
-</html>
+</html>
```

Comment:
Please add an empty line at the end of the file.
You can usually configure your editor to add the empty line on save, so you don't have to worry about it.